### ISSUE TITLE
[SUGGESTION]: Add Mega Stones to Pokemon Listings in Hatchery

### ISSUE THREAD

**Issue by Razorflamekun** [pokeclicker/pokeclicker#3866]
### Current Behavior

Mega Stones are not currently listed in the Pokemon's Box in the Hatchery.

### Improved Behavior

Acquired Mega Stones appear in the Pokemon's box in the Hatchery similar to how it appears in the Pokedex.

### Reason for change

There aren't enough places to view how much attack is required to mega evolve Pokemon in the game, and adding the Mega Stones to the Pokemon that have them in the Hatchery adds a much more visible alternative to using the Pokedex.

### Detailed Description

1. Click on Hatchery from the Shortcuts Menu.
2. Navigate to any Pokemon with a Mega Stone that you know has one.

Basically, just put the Mega Stone icon somewhere on the box for that Pokemon and when hovering over it, it either displays how much attack is required to Mega Evolve a Pokemon, or it will tell you that it is ready to Mega Evolve.

Thanks for reading!

**Comment by Farboo171** [comment]
UI improvements to the evo item modal resolve this issue.
